types/DIVAProtocolPolygon.md

```diff
@@ -73,19 +87,18 @@ keccak256(abi.encode("DIVAProtocolPolygon",abi.encode(1)))
 
 ### Encoding/Decoding
 
-A value of 99.9 would be submitted on-chain using the following bytes:
-
-`0x0000000000000000000000000000000000000000000000056a6418b505860000`
+For example, if the reference asset is ETH/USD with a price of $2,819.35, and the collateral token is DAI/USD at a price of $0.9996, then the hexadecimal of the bytes-encoded expected response for this query-- ready to be submitted by a reporter --is:
 
+`0x00000000000000000000000000000008490b0f122862a11da8c6ac40000000000000000000000000000000000000000000c08415c61401a79c97e10c00000000`
```

Comment:
it looks like these values were multiplied by 1e36, when they should have been multiplied by 1e18